test/Microsoft.AspNetCore.Hosting.FunctionalTests/ShutdownTests.cs

```diff
@@ -14,6 +14,8 @@ namespace Microsoft.AspNetCore.Hosting.FunctionalTests
     public class ShutdownTests
     {
         [ConditionalFact]
+        //Disabled until https://github.com/aspnet/Hosting/issues/974 resolved.
```

Comment:
Generally, it's better to disable using `[ConditionalFact(Skip = "Skip reason, url_to_issue")]` Easier than to remember to remove a OSSkipCondition when re-enabling the test, especially when there are more than one.